contrib/win32/win32compat/shell-host.c

```diff
@@ -39,6 +39,19 @@
 #include "misc_internal.h"
 #include "inc\utf.h"
 
+/* Terminal Types */
+#define TERM_TYPE_ANSI    "ansi"
+#define TERM_TYPE_VT100   "vt100"
+#define TERM_TYPE_XTERM   "xterm"
+
+#define TT_ANSI     0x0001        /* Codes in all ANSI like terminals. */
+#define TT_VT100    0x0002        /* VT100 */
+#define TT_XTERM    0x0004        /* Xterm */
+
+#define TTM_VT100X   (TT_VT100 | TT_XTERM)
```

Comment:
win32-openssh client scrolling is not working with your current changes.. Its a regression issue.
Please see my reply to the TERM setting below as a reply to another comment..